README.md

```diff
@@ -0,0 +1,87 @@
+# core
+A case management tool for the humanitarian sector - keep track of beneficiaries and services.
+
+## Architecture
+
+```
+├── api                             # Main Core go app
+│   ├── artifacts                   # docker files
+│   ├── cmd                         # main go package definition
+│   │   └── app
+│   └── pkg                         
+│       ├── apps                    # app modules    
+│       │   ├── cms             
+│       │   ├── iam
+│       │   ├── login
+│       │   │   └── templates
+│       │   ├── seed
+│       │   └── webapp              # client-facing web app
+│       │       └── templates
+│       ├── auth
+│       ├── middleware
+│       ├── rest
+│       ├── server                  # main server app
+│       ├── sessionmanager
+│       ├── testhelpers             # unused for now
+│       └── testing                 # backend unit tests
+└── e2e                 
+    └── cypress
+        ├── fixtures
+        ├── integration             # e2e tests
+        ├── plugins
+        └── support
+```
+
+## Getting started:
+
+1. Start the backend services:
+```bash
+cd core/api
```

Comment:
currently it is `core/api2`, will we move from `api2` to `api` ?

Reply:
Yeah, check the PR for COR-73